### PR TITLE
Add initial announcement for JuMP-dev 2023

### PR DESCRIPTION
**Pull request by odow**
We're getting a little squeezed for time because JuliaCon brought all of the dates forward to help people get visas in time. We can probably get an extension, or run a separate CFP if we run a separate event.

We also need a program committee, but I'll post on the Gitter once this is up.

### CODE REVIEW

**Comment by mlubin** [comment]
Do we really need to start taking submissions before we finalize the date and venue?

**Comment by odow** [comment]
I guess it's just how to work around the short window for JuliaCon submissions.

**Comment by odow** [comment]
If we do a separate event, no. And if we host at JuliaCon and the organizers are okay with us slipping in a bunch of late talks, no. But if we host at JuliaCon we'll need to confirm rooms etc when they begin to accept talks.

File: _posts/2022-11-20-jump_dev_2023.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Programme committee
+
+ * Oscar Dowson
+ * Chris Coey (RAI)
+ * Joaquim Dias Garcia (PSR)
+ * Benoît Legat (MIT)
+ * Miles Lubin (HRT)
+ * Joshua Pulsipher (CMU)
+
+To contact the program committee, write to [jump-dev-2023@googlegroups.com](mailto:jump-dev-2023@googlegroups.com).
```
Comment (mlubin) [inline _posts/2022-11-20-jump_dev_2023.md:67]: Mailto links seem very 1990s? They don't really work unless you have standalone email client.

File: _posts/2022-11-20-jump_dev_2023.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+title:  "JuMP-dev 2023"
+date:   2022-11-20
+categories: [jump-dev]
+permalink: /meetings/jumpdev2023/
+---
+
+We are pleased to announce that JuMP-dev 2023 will be co-located with
+[JuliaCon 2023](https://juliacon.org/2023)!
+
+This year, JuliaCon is being held 24-29 July, 2023 in Cambridge, Massachusetts.
```
Comment (mlubin) [inline _posts/2022-11-20-jump_dev_2023.md:12]: Maybe put additional emphasis that the conference will be held *in person*.

**Review by mlubin — APPROVED** [review]

File: _posts/2022-11-20-jump_dev_2023.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Speakers are encouraged to highlight challenges and lessons learned, not only
+successes. Talks should be aimed at a general audience, but familiarity with
+JuMP/Julia can be assumed.
+
+See the previous workshops for examples of accepted talks: [2017](/meetings/mit2017/),
+[2018](/meetings/bordeaux2018/), [2019](/meetings/santiago2019),
+[2021](/meetings/juliacon2021), and [2022](/meetings/juliacon2022).
+
+## Programme committee
```
Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:58]: @frapac @matbesancon @tjdiamandis do you want to be part of the program committee again?

Comment (frapac) [reply _posts/2022-11-20-jump_dev_2023.md:58]: With pleasure

**Comment by matbesancon** [comment]
Same here, with pleasure

File: _posts/2022-11-20-jump_dev_2023.md [hunk 4]
```diff
@@ -56,4 +55,12 @@ See the previous workshops for examples of accepted talks: [2017](/meetings/mit2
 [2018](/meetings/bordeaux2018/), [2019](/meetings/santiago2019),
 [2021](/meetings/juliacon2021), and [2022](/meetings/juliacon2022).
 
+## Programme committee
+
+ * Oscar Dowson
+ * Joaquim Dias Garcia (PSR)
+ * Benoît Legat (MIT)
```
Comment (blegat) [inline _posts/2022-11-20-jump_dev_2023.md:63]:
```suggestion
 * Benoît Legat (KU Leuven)
```

File: _posts/2022-11-20-jump_dev_2023.md (hunk 3 above)
Comment (matbesancon) [reply _posts/2022-11-20-jump_dev_2023.md:58]: would be happy to yes

Comment (tjdiamandis) [reply _posts/2022-11-20-jump_dev_2023.md:58]: Would love to!

**Comment by matbesancon** [comment]
> But if we host at JuliaCon we'll need to confirm rooms etc when they begin to accept talks

what if we ask to have the JuMP talks in one room and then consider we have the talk budget equivalent to that room in a single track?

**Review by pulsipher — APPROVED** [review]
This looks reasonable to me.

**Comment by odow** [comment]
> what if we ask to have the JuMP talks in one room and then consider we have the talk budget equivalent to that room in a single track?

Yes, I think this is the plan. Organizing committee is good with us co-locating. They will push some comms once we publish this.

**Comment by odow** [comment]
JuliaCon are going to extend the deadline until mid-January. There's a call tomorrow morning, so once finalized I'll merge this and then make a discourse post with the proper deadlines.

File: _posts/2022-11-20-jump_dev_2023.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+**To give a talk, [submit a proposal to JuliaCon](https://pretalx.com/juliacon2023/cfp)**
+**For "Session type," choose "Talk" or "Lightning talk." For "Track," choose**
+**"JuMP."**
+
+Similarly to [previous](/meetings/mit2017) [iterations](/meetings/bordeaux2018)
+[of](/meetings/santiago2019) [the](/meetings/juliacon2021) [workshop](/meetings/juliacon2022),
+we invite participants to present work related to JuMP. In particular, we
+especially seek talks related to:
+
+- JuMP core development ([MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl), JuMP 1.0)
```
Comment (mlubin) [inline _posts/2022-11-20-jump_dev_2023.md:40]: JuMP 2.0 instead of JuMP 1.0?

Comment (odow) [reply _posts/2022-11-20-jump_dev_2023.md:40]: 😆

**Pull request merged**